pkgs/development/compilers/lingua-franca/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, pkgs, stdenv, fetchFromGitHub, jdk11_headless, makeWrapper }:
+let
+  version = "0.1.0";
+in
+stdenv.mkDerivation {
+  name = "lfc";
+  version = version;
```

Comment:
```suggestion

stdenv.mkDerivation rec {
  pname = "lfc";
  version = "0.1.0";
```

Reply:
I did this because inside the derivation I couldn't find the ${version} variable.